tests/jenkins/run_as_user.sh

```diff
@@ -18,4 +18,5 @@ groupadd -f -g ${GROUP_ID} ${GROUP_NAME}
 useradd -m -u ${USER_ID} -g ${GROUP_NAME} ${USER_NAME}
 chown -R ${USER_NAME}:${GROUP_NAME} ${HOME_DIR}
 chown -R ${USER_NAME}:${GROUP_NAME} /usr/local/lib/
+echo "%${GROUP_NAME}  ALL=(ALL)       NOPASSWD: ALL" >> /etc/sudoers
```

Comment:
This script should only be called inside a docker container, so this is /etc/sudoers file of the container.